### DISCUSSION TITLE
Not seeing files following the "Regional simulation at Continental US domain using NLDAS2 forcing" Tutorial

### DISCUSSION THREAD

**Discussion by psbecker** [NCAR/hrldas#212] [Q&A]
I am trying to follow the [Note2 tutorial](https://github.com/NCAR/hrldas/blob/master/tutorial/Note2_2D_NLDAS2_domain.ipynb)
However, I'm not seeing many of the files that are supposedly there including the download.sh file and geo_em.d01_NLDAS0125.nc so not sure how to run the model without these. Does anyone know where these files might be?

**Comment by tslin2** [comment]
https://drive.google.com/drive/folders/1MR_4p7ke5BsOOspoITm10U3MNqy9mdeo

**Comment by psbecker** [comment]
Thank you!

Do you have an example of the download.sh shell code as well? I’m not seeing that in the drive

Best,
Paige

From: Tzu-Shun Lin ***@***.***>
Date: Saturday, September 14, 2024 at 11:32
To: NCAR/hrldas ***@***.***>
Cc: Paige Becker ***@***.***>, Author ***@***.***>
Subject: [EXTERNAL] Re: [NCAR/hrldas] Not seeing files following the "Regional simulation at Continental US domain using NLDAS2 forcing" Tutorial (Discussion #212)
CAUTION: This email originated from outside of the Colorado School of Mines organization. Do not click on links or open attachments unless you recognize the sender and know the content is safe.


https://drive.google.com/drive/folders/1MR_4p7ke5BsOOspoITm10U3MNqy9mdeo

**Reply by tslin2** [reply]
I do not have the shell code, but NLDAS2 is available on the website.
https://disc.gsfc.nasa.gov/information/alerts?title=NLDAS%20Version%20002%20(GRIB-1%20files)%20are%20no%20longer%20being%20updated%20after%2023Z%20on%20August%201,%202024

**Reply by mcthreems** [reply]
Hello,

The grib files are no longer available from this link. Is there another repository for them? Or is it possible to run the tutorial example using the newer netcdf files found at this link?

**Reply by tslin2** [reply]
Hi
yes, it is possible to run NLDAS netcdf files, but it will require some changes. At the same time, I can provide the grid files for this tutorial example.

**Reply by mcthreems** [reply]
I would appreciate it if you could provide the grib files for the tutorial case, would let me make sure things are working properly. As for using the netcdf files, I assume I would need those for any future simulations I want to set up? How substantial would the needed changes be to accommodate that?

**Reply by cenlinhe** [reply]
We are updating our HRLDAS_forcing preprocessor to handle the netcdf NLDAS-2 files, which should be ready in the next few weeks.

**Reply by tslin2** [reply]
You can find the grid files in the below for this test case:
https://drive.google.com/drive/folders/1MJeR5OlWP54MmfvHwmM9Uw0lhFdUPJ-j?usp=sharing

**Reply by mcthreems** [reply]
Thank you. I followed the link and submitted an access request.

**Comment by psbecker** [comment]
Thank you!

From: Tzu-Shun Lin ***@***.***>
Date: Monday, September 16, 2024 at 11:59
To: NCAR/hrldas ***@***.***>
Cc: Paige Becker ***@***.***>, Author ***@***.***>
Subject: [EXTERNAL] Re: [NCAR/hrldas] Not seeing files following the "Regional simulation at Continental US domain using NLDAS2 forcing" Tutorial (Discussion #212)
CAUTION: This email originated from outside of the Colorado School of Mines organization. Do not click on links or open attachments unless you recognize the sender and know the content is safe.


I do not have the shell code, but NLDAS2 is available on the website.
https://disc.gsfc.nasa.gov/information/alerts?title=NLDAS%20Version%20002%20(GRIB-1%20files)%20are%20no%20longer%20being%20updated%20after%2023Z%20on%20August%201,%202024

**Comment by tslin2** [comment]
You're very welcome.
